src/pages/instances/actions/AttachIsoBtn.tsx

```diff
@@ -46,14 +47,19 @@ const AttachIsoBtn: FC<Props> = ({ instance }) => {
         const instanceLink = instanceLinkFromOperation({
           operation,
           project,
+          instanceType: instance.type,
         });
         eventQueue.set(
           operation.metadata.id,
           () =>
             toastNotify.success(
               <>
-                ISO <b>{attachedIso?.source ?? ""}</b> detached from{" "}
-                {instanceLink}
+                ISO{" "}
+                <ResourceLabel
+                  type="iso-volume"
+                  value={attachedIso?.source ?? ""}
+                />{" "}
```

Comment:
Should be bold after merging the new resource label. Applies to probably all other cases where we use the label in a notification.

Reply:
yes exactly, that was the first task after I rebased main :)